### ISSUE TITLE
use [u8; 64] for signature

### ISSUE THREAD

**Issue by goastler** [prosopo/protocol#382]

**Comment by goastler** [comment]
Currently, we use two `[u8; 32]` arrays in the contract to house a signature. This should be a `[u8; 64]` but ink did not support this until lately (https://github.com/paritytech/ink/pull/1787). Upgrade ink to get this feature then implement it

**Comment by goastler** [comment]
Blocked by the sr25519 signature verification. This needs to be merged into ink before we can upgrade https://github.com/paritytech/ink/pull/1757

**Comment by goastler** [comment]
also need to add the `Copy` trait back in when this change has been made
